Projects/Presentation/Sources/Home/View/Component/RoutineDeleteAlertView.swift

```diff
@@ -0,0 +1,116 @@
+    }
+
+    private let contentView = UIView()
+    private let deleteLabel = UILabel()
+    private let deleteDailyRoutineButton = UIButton()
+    private let deleteAllRoutineButton = UIButton()
+    weak var delegate: RoutineDeleteAlertViewDelegate?
+
+    init() {
+        super.init(frame: .zero)
+        configureAttribute()
+        configureLayout()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    private func configureAttribute() {
+        contentView.backgroundColor = .white
+        contentView.layer.masksToBounds = true
+        contentView.layer.cornerRadius = 20
+
+        deleteLabel.text = "해당 루틴은\n반복 루틴으로 설정되어 있어요"
+        deleteLabel.numberOfLines = 2
+        deleteLabel.textAlignment = .center
+        deleteLabel.font = BitnagilFont(style: .body1, weight: .semiBold).font
+        deleteLabel.textColor = BitnagilColor.gray10
+
+        var deleteDailyRoutineButtonConfiguration = UIButton.Configuration.filled()
+        deleteDailyRoutineButtonConfiguration.baseBackgroundColor = .white
+        deleteDailyRoutineButtonConfiguration.background.cornerRadius = 8
+        deleteDailyRoutineButtonConfiguration.attributedTitle = AttributedString(
+            "당일만 삭제",
+            attributes: .init([.font: BitnagilFont(style: .body2, weight: .medium).font]))
+        deleteDailyRoutineButtonConfiguration.baseForegroundColor = BitnagilColor.navy500
+        deleteDailyRoutineButtonConfiguration.background.strokeColor = BitnagilColor.navy500
+        deleteDailyRoutineButtonConfiguration.background.strokeWidth = 1
+        deleteDailyRoutineButton.configuration = deleteDailyRoutineButtonConfiguration
+        deleteDailyRoutineButton.addAction(UIAction { [weak self] _ in
+            guard let self else { return }
+            self.delegate?.routineDeleteAlertViewDidTapDeleteDailiyRoutine(self)
+        }, for: .touchUpInside)
+
+        var deleteAllRoutineButtonConfiguration = UIButton.Configuration.filled()
+        deleteAllRoutineButtonConfiguration.baseBackgroundColor = .white
+        deleteAllRoutineButtonConfiguration.background.cornerRadius = 8
+        deleteAllRoutineButtonConfiguration.attributedTitle = AttributedString(
+            "전체 루틴 삭제",
+            attributes: .init([.font: BitnagilFont(style: .body2, weight: .medium).font]))
+        deleteAllRoutineButtonConfiguration.baseForegroundColor = BitnagilColor.navy500
+        deleteAllRoutineButtonConfiguration.background.strokeColor = BitnagilColor.navy500
+        deleteAllRoutineButtonConfiguration.background.strokeWidth = 1
+        deleteAllRoutineButton.configuration = deleteAllRoutineButtonConfiguration
+        deleteAllRoutineButton.addAction(UIAction { [weak self] _ in
+            guard let self else { return }
+            self.delegate?.routineDeleteAlertViewDidTapDeleteAllRoutine(self)
+        }, for: .touchUpInside)
```

Comment:
```suggestion
        deleteAllRoutineButton.addAction(
            UIAction { [weak self] _ in
                guard let self else { return }
                self.delegate?.routineDeleteAlertViewDidTapDeleteAllRoutine(self)
            },
            for: .touchUpInside)
```

정말 사소한 부분이긴하지만, 요런 형식이 컨벤션에 맞는것 같습니다!